docs/content/latest/api/ysql/the-sql-language/statements/cmd_analyze.md

```diff
@@ -0,0 +1,139 @@
+---
+title: ANALYZE statement [YSQL]
+headerTitle: ANALYZE
+linkTitle: ANALYZE
+description: Collect statistics about database tables with the ANALYZE statement.
+beta: /latest/faq/general/#what-is-the-definition-of-the-beta-feature-tag
```

Comment:
**Questions:**

1. Which version supports ANALYZE on non-temporary tables?
2. Do you want to wait to publish this in conjunction with that release?

No matter what, we should add a note before the examples mentioning which versions support analyzing non-temporary tables, since the current latest release doesn't.

**Using the 2.7.2 release**, I launched a YugabyteDB instance with `bin/yugabyted start --tserver_flags="ysql_beta_features"`, and followed the "Analyze affects query plans" example.

Things went as expected for table creation, population, and the first EXPLAIN. But `ANALYZE test` gave the following output:

```
WARNING:  analyzing non-temporary tables will be ignored
ANALYZE
```

...and my query plan didn't change:

```
                       QUERY PLAN                        
---------------------------------------------------------
 Seq Scan on test  (cost=0.00..102.50 rows=1000 width=8)
   Filter: (b = 1)
(2 rows)
```

And `ANALYZE` with no table specified gave me 71 lines of this:

```
WARNING:  analyzing non-temporary tables will be ignored
```

Reply:
The feature is introduced by 638f4188da8b3f81a595eea7b7e45f2ed97104ab, I believe it will be released with 2.7.3, though I am not sure about release schedule. It is currently available in master.
What is our practice so far? I do not see any reason to make an exception for this feature.

Reply:
I wouldn't publish this until it's actually available. Let's get this all ready to go, but not merge until 2.7.3 is going out.

Reply:
OK, works for me.